nautobot/dcim/filters.py

```diff
@@ -1541,13 +1540,17 @@ class Meta:
 class CableFilterSet(NautobotFilterSet, StatusModelFilterSetMixin):
     q = SearchFilter(filter_predicates={"label": "icontains"})
     type = django_filters.MultipleChoiceFilter(choices=CableTypeChoices)
-    color = django_filters.MultipleChoiceFilter(choices=ColorChoices)
+    color = MultiValueCharFilter()
     device_id = MultiValueUUIDFilter(method="filter_device", label="Device (ID)")
     device = MultiValueCharFilter(method="filter_device", field_name="device__name", label="Device (name)")
     rack_id = MultiValueUUIDFilter(method="filter_device", field_name="device__rack_id", label="Rack (ID)")
     rack = MultiValueCharFilter(method="filter_device", field_name="device__rack__name", label="Rack (name)")
     site_id = MultiValueUUIDFilter(method="filter_device", field_name="device__site_id", label="Site (ID)")
     site = MultiValueCharFilter(method="filter_device", field_name="device__site__slug", label="Site (name)")
+    region_id = MultiValueUUIDFilter(method="filter_device", field_name="device__site__region_id", label="Region (ID)")
+    region = MultiValueCharFilter(
+        method="filter_device", field_name="device__site__region__slug", label="Region (name)"
+    )
```

Comment:
Can we instead just add `region` as a `NaturalKeyOrPKMultipleChoiceFilter`?

Reply:
Im not sure it is possible to use `NaturalKeyOrPKMultipleChoiceFilter`. As you can see from other filters in this filterset, we have to use a custom method `filter_device` to filter the regions for both`termintion_a` and `termination_b`.  In `NaturalKeyOrPKMultipleChoiceFilter` I can only pass in one or the other (`_termination_a_device__site__region`, `_termination_b_device__site__region`) for its `field_name` argument. Unless Im missing something obvious🤔